features/cxf/src/main/feature/feature.xml

```diff
@@ -339,7 +339,7 @@
     <feature name="cxf-jsr-json" version="${cxf.version}">
         <bundle dependency="true" start-level="30">mvn:org.apache.aries.spifly/org.apache.aries.spifly.dynamic.bundle/${cxf.aries.fly.version}</bundle>
         <bundle dependency="true" start-level="35">mvn:org.apache.geronimo.specs/geronimo-json_1.1_spec/${cxf.geronimo.json.version}</bundle>
-        <bundle start-level="35">mvn:org.apache.johnzon/johnzon-core/${cxf.johnzon.version}</bundle>
+<!--        <bundle start-level="35">mvn:org.apache.johnzon/johnzon-core/${cxf.johnzon.version}</bundle>-->
```

Comment:
❔ should this just be removed?

Reply:
It probably could, but if you look through the rest of this file there's a bunch of other commented-out ones that were left rather than deleting them so I kept it to be consistent